### PR TITLE
add ADEL space

### PR DESCRIPTION
**Pull request by AlexanderMazaletskiy**

### CODE REVIEW

File: spaces/akropolis-delphi/index.json [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+    "0x5fFcf7da7BdC49CA8A2E7a542BD59dC38228Dd45",
+    "0x73fC3038B4cD8FfD07482b92a52Ea806505e5748",
+    "0x3501Ec11d205fa249f2C42f5470e137b529b35D0",
+    "0x1A547c3dd03c39Fb2b5aEaFC524033879bD28F13",
+    "0xF311b1258d0F245b85090e4Fb01f2277cB2328aD"
+  ],
+  "min": 0,
+  "invalid": [],
+  "strategies": [
+    ["balancer", {
```
Comment (bonustrack) [inline spaces/akropolis-delphi/index.json:19]: If you want to use BPT to score proposal results that's not the way to do it. You could count ADEL and ADEL inside BPT pools if you want. You will need to use erc20BalanceOf + balancer strategies with `0x94d863173EE77439E4292284fF13fAD54b3BA182` as address on both strategies

**Comment by AlexanderMazaletskiy** [comment]
Our idea is that tokens are put in proportions to the user in the Balancer Pool. And the tokens received by BPT are already participating in the voting.

**Comment by AlexanderMazaletskiy** [comment]
@bonustrack Please check

**Comment by e-asia** [comment]
@AlexanderMazaletskiy 
The colors used in your skin make the interface unreadable, please change with others or remove the skin to use the default theme.
image

**Comment by AlexanderMazaletskiy** [comment]
@e-asia Please, check

**Review by e-asia — APPROVED** [review]

File: spaces/akropolis-delphi/index.json [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+    "0x4C39b37f5F20a0695BFDC59cf10bd85a6c4B7c30",
+    "0x5fFcf7da7BdC49CA8A2E7a542BD59dC38228Dd45",
+    "0x73fC3038B4cD8FfD07482b92a52Ea806505e5748",
+    "0x3501Ec11d205fa249f2C42f5470e137b529b35D0",
+    "0x1A547c3dd03c39Fb2b5aEaFC524033879bD28F13",
+    "0xF311b1258d0F245b85090e4Fb01f2277cB2328aD"
+  ],
+  "min": 0,
+  "invalid": [],
+  "strategies": [
```
Comment (bonustrack) [inline spaces/akropolis-delphi/index.json:18]: If you want to use the BPT you should remove strategies array, and change `address` and `token` to the BPT address. Also i recommend to use symbol "ADEL-AKRO"

File: spaces/akropolis-delphi/index.json (hunk 1 above)
Comment (AlexanderMazaletskiy) [reply spaces/akropolis-delphi/index.json]: @bonustrack @e-asia Please, check

File: spaces/akropolis-delphi/index.json [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+    "0x4C39b37f5F20a0695BFDC59cf10bd85a6c4B7c30",
+    "0x5fFcf7da7BdC49CA8A2E7a542BD59dC38228Dd45",
+    "0x73fC3038B4cD8FfD07482b92a52Ea806505e5748",
+    "0x3501Ec11d205fa249f2C42f5470e137b529b35D0",
+    "0x1A547c3dd03c39Fb2b5aEaFC524033879bD28F13",
+    "0xF311b1258d0F245b85090e4Fb01f2277cB2328aD"
+  ],
+  "min": 0,
+  "invalid": []
+}
```
Comment (e-asia) [inline spaces/akropolis-delphi/index.json:18]: We have updated the format in `index.json`. 
Please add 2 new parameters `chainId` and `decimals`.
You can refer to the documentation that has been completed. https://docs.snapshot.page/guides/create-a-space#3-change-your-space-metadata

**Review by e-asia — CHANGES_REQUESTED** [review]

File: spaces/homepage.json [hunk 4]
```diff
@@ -8,5 +8,6 @@
   "rarible",
   "pickle",
   "aavegotchi",
-  "uniswap"
+  "uniswap",
+  "akropolis-delphi"
```
Comment (e-asia) [inline spaces/homepage.json:12]: To be approved, your space will need to be active with few proposals and significant participation rate.

**Review by e-asia — CHANGES_REQUESTED** [review]
The images file size should not exceed 50KB. Please reduce their weight.

**Comment by e-asia** [comment]
@AlexanderMazaletskiy You should also delete `logo1.png` to add only `logo.png`. Thank you.

File: spaces/homepage.json (hunk 4 above)
Comment (in19farkt) [reply spaces/homepage.json]: Fixed

**Comment by in19farkt** [comment]
> You should also delete logo1.png to add only logo.png. Thank you.
@e-asia But `logo1.png` needed for the second strategy, or not?

**Comment by bonustrack** [comment]
@AlexanderMazaletskiy Yes you should keep that file for the second strategy.

**Comment by in19farkt** [comment]
> The images file size should not exceed 50KB. Please reduce their weight.

Fixed.

**Comment by AlexanderMazaletskiy** [comment]
For strategy, you need check this PR https://github.com/bonustrack/snapshot.js/pull/4

**Pull request merged**